release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3488,3 +3488,22 @@ Now these IP addresses are periodically collected and made available to be reall
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+[id="ocp-4-6-56"]
+=== RHBA-2022:0867 - {product-title} 4.6.56 bug fix and security update
+
+Issued: 2022-03-23
+
+{product-title} release 4.6.56, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0867[RHBA-2022:0867] advisory. There are no RPM packages for this release. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0866[RHSA-2022:0866] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6833311[{product-title} 4.6.56 container image list]
+
+[id="ocp-4-6-56-bug-fixes"]
+==== Bug fixes
+ * Previously, an error between {rh-openstack} credentials secret creation and `kube-controller-manager` startup prevented {rh-openstack} credentials from configuring. This caused the `LoadBalancer` services to not be created in {rh-openstack}. This fix fetches the {rh-openstack} credentials during the `kube-controller-manager` startup. This results in a routinely initialized {rh-openstack} cloud provider during the startup of the `kube-controller-manager`. (link:https://kbugzilla.redhat.com/show_bug.cgi?id=2059677[*BZ#2059677*])
```

Comment:
from configuring properly? from properly configuring? I like the former. Here's a small suggestion to reduce wordiness: 

 * Previously, an error between {rh-openstack-first} credentials secret creation and `kube-controller-manager` startup prevented {rh-openstack} secret credentials from configuring properly. Consequently, `LoadBalancer` services were not created in {rh-openstack}. This update fetches the {rh-openstack} secret credentials when `kube-controller-manager` starts. As a result,  {rh-openstack} secret credentials now initialize properly. (link:https://kbugzilla.redhat.com/show_bug.cgi?id=2059677[*BZ#2059677*])

Reply:
Thanks for this suggestion! It is helpful, and I'll use this as an example when working on my sentence structure! Update made. :)